docs/migrating_versions/migrating_to_v050.rst

```diff
@@ -0,0 +1,89 @@
+.. _migrate_to_050:
+
+Migrating to v0.5.0 from v0.4.6
+===============================
+
+The update from v0.4.6 of Fairlearn has brought some major changes. This
```

Comment:
Moments had rather massive changes. Do you want me to outline those or do you want to add them? It's mainly about the separation of epsilon into epsilon on `ExponentiatedGradient` and the bounds on the moments. We've preserved the same behavior of you don't specify the bound (or previously epsilon), so that makes it at least a bit less painful.

Reply:
Could you send a slightly more verbose summary, and I'll work it in. You're rather more familiar with that portion of the code than I am.

Reply:
Just added something. Please read carefully though, since I was just working through the Changes.md.

Reply:
Added a few comments. This is great!